packages/vue/src/scroll-area/ScrollAreaCornerImpl.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import { computed, ref } from 'vue'
+import { useScrollAreaContext } from './ScrollArea'
+import { SCROLL_AREA_CORNER_IMPL_NAME, SCROLL_AREA_CORNER_NAME } from './constants'
+import type { ScrollAreaCornerImplProps } from './ScrollAreaCornerImpl'
+import { Primitive } from '@oku-ui/primitive'
+import { useResizeObserver } from '@vueuse/core'
```

Comment:
```suggestion
import { useResizeObserver } from '@oku-ui/use-composable'
```

https://github.com/oku-ui/primitives/blob/main/packages/vue/src/use-composable/index.ts#L1

please export `useResizeObserver`

Reply:
fixed